test/TestExpectations.json

```diff
@@ -545,6 +545,12 @@
     "parameters": ["webDriverBiDi"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[defaultbrowsercontext.spec] DefaultBrowserContext page.cookies() should work",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["webDriverBiDi"],
+    "expectations": ["PASS"]
```

Comment:
Let's inverse the expectation? Remove the expectation on line519 and mark failing as FAIL?

Reply:
Or are too many still failing?

Reply:
Tried it, there are still a lot of failures from Firefox